gallery/how_to/work_with_microtvm/install_zephyr.py

```diff
@@ -0,0 +1,29 @@
+%%shell
```

Comment:
Probably it would be better to format this script as a documentation file in sphinx format?

Reply:
These are now reStructuredText files.